Licensing: add Migration guide + some fixes (#8843)
DevExpress/devextreme-documentation: concepts/Common/Licensing/00 Licensing.md

You must purchase a license to use DevExtreme UI components/libraries within a software project. If you are ready to incorporate DevExtreme UI components/libraries in your project, please visit the [DevExtreme Purchase](https://js.devexpress.com/Buy/) page for pricing options.

> [note] For more information on product licensing and the terms that govern use of DevExtreme UI components/libraries, visit the following webpage: [DevExtreme Licensing](https://js.devexpress.com/Licensing/).

Reviewer: ```suggestion
[note] For more information about product licensing and the terms that govern the use of DevExtreme UI components and libraries, see [DevExtreme Licensing](https://js.devexpress.com/Licensing/).
```

> Please remember that Developer Express Inc licenses DevExtreme on a per developer basis. Each developer using DevExtreme must purchase/acquire a license. If your organization owns multiple licenses, you can assign individual licenses to different developers. By assigning available licenses to developers within your organization, each developer will have access to their own personal license key within the DevExpress Download Manager (under their personal DevExpress account).

Reviewer: ```suggestion
Developer Express Inc. licenses DevExtreme on a per-developer basis. Each developer who uses DevExtreme must purchase or obtain a license. If your organization owns multiple licenses, you can assign them to individual developers. After a license is assigned, the developer can access a personal license key in the DevExpress Download Manager through their DevExpress account.
```

> If a developer leaves or stops using DevExtreme, you can revoke their license and reassign it to another individual.

Reviewer: ```suggestion
If a developer leaves or stops using DevExtreme, you can revoke their license and reassign it.
```

> DevExpress checks the license status to determine whether a valid DevExtreme license exists. If you do not have a valid DevExtreme license, an error message is displayed on-screen. To complete the license verification process, you must [supply a valid license key](/concepts/Common/Licensing/10%20Validate%20Your%20License%20Key.md '/Documentation/Guide/Common/Licensing/#Validate_Your_License_Key').

Reviewer: ```suggestion
DevExpress checks the license status to determine whether a valid DevExtreme license is available. If you do not have a valid DevExtreme license, an error message is displayed. To complete the license verification process, you must [supply a valid license key](/concepts/Common/Licensing/10%20Validate%20Your%20License%20Key.md '/Documentation/Guide/Common/Licensing/#Validate_Your_License_Key').
```

You must purchase a license to use DevExtreme UI components/libraries within a software project. If you are ready to incorporate DevExtreme UI components/libraries in your project, please visit the [DevExtreme Purchase](https://js.devexpress.com/Buy/) page for pricing options.

[note] For more information on product licensing and the terms that govern use of DevExtreme UI components/libraries, visit the following webpage: [DevExtreme Licensing](https://js.devexpress.com/Licensing/).

Please remember that Developer Express Inc licenses DevExtreme on a per developer basis. Each developer using DevExtreme must purchase/acquire a license. If your organization owns multiple licenses, you can assign individual licenses to different developers. By assigning available licenses to developers within your organization, each developer will have access to their own personal license key within the DevExpress Download Manager (under their personal DevExpress account).

> If a developer leaves or stops using DevExtreme, you can revoke their license and reassign it to another individual.

Reviewer 2: I'm sorry, I ment that we can remove only mentions 23.2+
- change or remove paragraph's caption
- remove text 'With the release of v23.2, '
Because 23.2 was too long ago to still mention it.

Everything else is totally ok for me here.
